src/test/java/seedu/rc4hdb/logic/commands/modelcommands/ModelCommandTestUtil.java

```diff
@@ -16,11 +16,11 @@
 import seedu.rc4hdb.commons.core.index.Index;
 import seedu.rc4hdb.logic.commands.CommandResult;
 import seedu.rc4hdb.logic.commands.exceptions.CommandException;
-import seedu.rc4hdb.logic.commands.modelcommands.EditCommand.EditPersonDescriptor;
+import seedu.rc4hdb.logic.commands.modelcommands.EditCommand.EditResidentDescriptor;
```

Comment:
Same issue as above: the nested class `EditResidentDescriptor` has been abstracted out into `ResidentDescriptor` and does not exist within `EditCommand`.